include/SDL_events.h

```diff
@@ -325,14 +325,12 @@ typedef struct SDL_PanEvent
     Uint32 type;          /**< ::SDL_PANEVENT */
     Uint32 timestamp;   /**< In milliseconds, populated using SDL_GetTicks() */
     Uint32 windowID;    /**< The window with mouse focus, if any */
-    Uint64 x;      /**< Precise scrolling amount on x axis. */
-    Uint64 y;      /**< Precise scrolling amount on y axis. */
     Uint32 which;       /**< The mouse instance id, or SDL_TOUCH_MOUSEID */
-    Uint32 source_type; /**< One of SDL_MOUSEWHEEL_SOURCE_[...] */
-    Uint8  contains_x; /**< Indicates event contains a useful value in scalar_x and a pan should be calculated */
-    Uint8  contains_y; /**< Indicates event contains a useful value in scalar_y and a pan should be calculated */
-    Uint8  interrupt;  /**< If some fling event was dispatched, this is intended to terminate it */
-    Uint8  fling;      /**< Indicates the user has "flung" the wheel and kinetic scrolling (if enabled) should begin here */
+    SDL_PanAxis axis;
+    SDL_PanType pantype;
+    SDL_PanContents contents;
```

Comment:
What will `contents` be for `pantype == SDL_PANEVENTTYPE_SOURCE`

Reply:
Currently nothing, as that event is not broadcast. If it were, the contents would probably be an empty SDL_PanType_Source struct similar to Interrupt and Fling events

Reply:
Alright cool! That's the only questionable thing I could find, although that's not much of an assurance!